.github/workflows/docs-release.yml

```diff
@@ -21,7 +21,11 @@ jobs:
           if [ -z ${{ github.event.inputs.version }} ]; then echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV; else echo "RELEASE_VERSION=${{ github.event.inputs.version }}" >> $GITHUB_ENV; fi
       - name: Release 🌱
         run: |
-          cd docs/source && pip install -r requirements.txt
+          python -m pip install --upgrade pip
+          python -m pip install poetry
+          pushd core; poetry export -f requirements.txt --output ../docs/source/requirements_core.txt --without-hashes; popd
+          pushd cli; poetry export -f requirements.txt --output ../docs/source/requirements_cli.txt --without-hashes; pushd
```

Comment:
Is it normal that this command ends with `pushd` instead of `popd`?

Reply:
Definitely not..